src/utils/number.js

```diff
@@ -229,7 +228,7 @@ export function format (value, options) {
 
     default:
       throw new Error('Unknown notation "' + notation + '". ' +
-          'Choose "auto", "exponential", or "fixed".')
+        'Choose "auto", "exponential", or "fixed".')
```

Comment:
VSCode changed spacing there... Should I undo these type of edits?

Reply:
There is a linter in place, you can run the following to see if there are linter errors:

```
npm run lint
```

and you can run the following to automatically fix most of the linting issues:
```
npm run lint -- --fix
```

Nicest would be to get your VSCode code style settings and the mathjs linting rules aligned 😄 .

Reply:
I ran the linter, but it seems VSCode "corrected" these whitespaces and the linter does not care. 

Do you want these reverted?

Reply:
If the linter is happy it's all fine. If we feel like we need the linter to be configured more strictly in this regard we can do so, though let's discuss in a separate issue in that case.